fdbserver/DataDistribution.actor.cpp

```diff
@@ -3080,7 +3115,13 @@ ACTOR Future<Void> storageServerTracker(
 
 			state bool recordTeamCollectionInfo = false;
 			choose {
-				when( wait( failureTracker ) ) {
+				when(bool ignoreSSFailures = wait(failureTracker)) {
+					if (ignoreSSFailures) {
```

Comment:
Did you test that you switch on and off the `ignoreSSFailures ` for multiple times?

If you disable the stoageServerTracker when you enable ignoreSSFailures, do you enable the tracker when you disable the ignoreSSFailures? If not, this should be a problem because SS status will no longer be tracked.

Reply:
`when ( wait( self->healthyZone.onChange() ) ) {}` will make sure when the switch status changed via CLI, the `storageServerFailureTracker` loop will be rerun and respective heath change monitors will be re-enabled.